app/views/submissions/_form.html.haml

```diff
@@ -9,13 +9,10 @@
   %i.fa.fa-question-circle.helpbox{"data-toggle" => "tooltip", :title => "Sekcja 'Finansowanie' jest przeznaczona do umieszczenia informacji kto finansował badania opublikowane w artykule."}
   = f.input :funding, label: "Finansowanie"
   = f.input :remarks, label: "Uwagi edytora"
-  %h3 Język polski
-  = f.input :polish_title, label: "Tytuł"
-  = f.input :polish_abstract, label: "Streszczenie"
-  = f.input :polish_keywords, label: "Słowa kluczowe"
+  = f.input :polish_title, label: "Tytuł", include_blank: false, label_method: :title
   %h3 Język angielski
-  = f.input :english_title, label: "Title"
-  = f.input :english_abstract, label: "Abstract"
-  = f.input :english_keywords, label: "Key words"
+  = f.input :english_title, label: "Title", include_blank: false, label_method: :title
+  = f.input :english_abstract, label: "Abstract", include_blank:false
+  = f.input :english_keywords, label: "Key words", include_blank:false
```

Comment:
W trzech linijkach wyżej proszę usunąć `include_blak: false` - to ma sens tylko w przypadku pola typu `select`.